station/recipes/meteor_qpsk.py

```diff
@@ -0,0 +1,78 @@
+from contextlib import suppress
+from datetime import timedelta
+import os.path
+import signal
+
+import sh
+
+from recipes.helpers import set_sh_defaults
+
+
+@set_sh_defaults
+def execute(working_dir: str, frequency: str, duration: timedelta, sh=sh):
+    signal_path = os.path.join(working_dir, "signal.wav")
+    product_path = os.path.join(working_dir, "product.png")
+
+    normalized_signal_path = os.path.join(working_dir, "normalized_signal.wav")
+    qpsk_path = os.path.join(working_dir, "qpsk")
+    dump_prefix_path = os.path.join(working_dir, "dump")
+    dump_path = dump_prefix_path + ".dec"
+    product_raw_prefix_path = os.path.join(working_dir, "product_raw")
+    product_raw_path = product_raw_prefix_path + ".bmp"
+
+    # Record signal
+    rtl_fm_proc = sh.rtl_fm(
+        "-M", "raw",
```

Comment:
Those parameters should be documented. But given that we're going to migrate to gnu radio soon, perhaps there's no point in investing too much time in this... OTOH, this may make the migration slightly easier. Anyway, it's your call. Feel free to ignore this.

Reply:
I don't know why some parameters are like that. I just analyze examples in Internet and perform few experiments with GQRX, but I try.

Reply:
I described the value using my best known.

Reply:
Great!